source/Junkership.js

```diff
@@ -36,7 +36,7 @@ export default class Junkership extends Pixi.Sprite {
             RapidSprayShot, SprayShot, SuperSprayShot,
             CrazySprayShot, VertSprayShot, VertShoota,
             BFG, Laser, PiercingLaser, SuperLaser, Mine,
-            SuperMine, PaintShot, PeaShoota]
+            SuperMine, PaintShot, Sword, PeaShoota]
```

Comment:
As part of removing dependencies, `this.WeaponList` isn't actually used at all, and was removed earlier. You can remove it  again with no issues.